src/main/scala/com/fulcrumgenomics/testing/VariantContextSetBuilder.scala

```diff
@@ -129,6 +131,7 @@ class VariantContextSetBuilder(sampleNames: Seq[String] = List("Sample")) extend
       case gAlleles => new GenotypeBuilder(name, toAlleles(gAlleles, referenceAllele=referenceAllele).asJava)
     }
     genotypeBuilder.phased(phased)
```

Comment:
Do you want to make sure that the length of alleleDepths (when >0) is the same as the # of alleles?